app/views/recommendations/review_form.html.erb

```diff
@@ -29,12 +29,14 @@
     class: "govuk-link"
   ) %>
 
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
   <%= render(
     "recommendations/events",
-    { recommendations: @planning_application.recommendations }
+    { recommendations: @recommendations }
   ) %>
 
-  <%= form_for @recommendation, url: review_planning_application_path(@planning_application) do |form| %>
+  <%= form_for @recommendation, url: review_planning_application_recommendation_path(@planning_application, @recommendation) do |form| %>
```

Comment:
While you're here it might be worth updating this form to use [the gov uk form builder gem](https://govuk-form-builder.netlify.app/introduction/get-started/)? But it's a faff, so happy for use to leave it for another day!